C/services/common/plugin_manager.cpp

```diff
@@ -399,6 +400,10 @@ char		buf[MAXPATHLEN];
 	{
 		pluginHandle = new NotificationPythonPluginHandle(name.c_str(), buf);
 	}
+	else if (type.compare("filter") == 0)
```

Comment:
plugin_api.h has defines for the filter type strings, it would be nicer to use these.